Library/Formula/bazel.rb

```diff
@@ -20,6 +20,8 @@ def install
       "/etc/bazel.bazelrc",
       "#{etc}/bazel/bazel.bazelrc"
 
+    ENV["EMBED_LABEL"] = "#{version}-homebrew"
```

Comment:
Also compile.sh knows nothing about where it was built on. Here we mark the binary as being built by homebrew.